src/index.js

```diff
@@ -54,8 +54,8 @@ export default function mitt(all: EventHandlerMap) {
 		 * @memberOf mitt
 		 */
 		emit(type: string, evt: any) {
-			(all[type] || []).slice().map((handler) => { handler(evt); });
-			(all['*'] || []).slice().map((handler) => { handler(type, evt); });
+			(all[type] || []).forEach((handler) => { handler(evt); });
```

Comment:
By doing this the library will drop the support for IE9 as it does not support `forEach` while this might be a saving in size and allow for some performance change (marginal in most real cases) dropping support for IE 9 will be an issue.